PokedexApp/PokedexApp/Controllers/Tabs/PokedexViewController.swift

```diff
@@ -72,16 +77,20 @@ class PokedexViewController: UIViewController {
     override func viewDidLayoutSubviews() {
         super.viewDidLayoutSubviews()
         
-        setupLayoutSearchBar()
+        setupLayoutSearchBar(height: 40)
         setupLayoutCollectionView()
         
         view.addSubview(textNotification)
         textNotification.frame = view.bounds
-
     }
     
     // MARK: - Setup method
-
+    static func createPokedexViewController(nameType: String) -> PokedexViewController {
```

Comment:
e cho func này xuống cuối cùng nhé
```suggestion
    static func instance(nameType: String) -> PokedexViewController {
```